### PR TITLE
Update ESkae011.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Description for Summer School. 
The description is not full (reconstructed after I lost it Saturday night, unfortunately). I included some Information for the cataloguing leaders who will do this MS with their group, and who will need an updated description. It is a complex MS, it requires further work.

### CODE REVIEW

File: ES/ESkae011.xml [hunk 1]
```diff
@@ -130,11 +111,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                         <placeName ref="INS0234KAE"/>
                      </origPlace>
-                     <persName ref="PRS9451Tewoflos">
-                        <roleName type="office">ʾǝč̣č̣age</roleName> Tewoflos </persName> was in
-                     office in 
-                     <origDate from="1872" to="1889" evidence="internal">
-                      1872-1889</origDate>
+                     <origDate from="1840" to="1889" evidence="internal">Around the middle of the 19th century or somewhat
+                       later but  before 1889.</origDate>
+                      The commissioner and owner of <ref target="#p1">Unit</ref> was <persName ref="PRS9451Tewoflos" role="owner">mamhǝr Gabra Wald</persName>; the main commissioner of
```
Comment (thea-m) [inline ES/ESkae011.xml:116]: Tewoflos or Gabra Wald?

File: ES/ESkae011.xml [hunk 2]
```diff
@@ -272,6 +270,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        fetch the missive in the book.</desc>
+                                 <q xml:lang="am"><seg part="I">ሃይማኖት፡ ማለት፡ አዋን፡ ብሎ፡ የሚቀበሉት፡ ኡነት፡ ብለው፡ የሚፀኑበት፡ ማለት᎓
+                                    ነው፡ ጦማረ፡ ሃይማኖት፡ ዘፈነዋ፡ እጨጌ፡ ቴዎፍሎስ፡ ዘነበረ፡ በመንበረ፡ ተክለ፡ ሃይማኖት፡ ውስተ፡ ኰሎን፡ አብያተ፡ ክርስቲያናት፡
+                                  ኀገረ፡ ተንቤን። በእዴሁ፡ ለአቡነ፡ መምሕር፡ ገብረ፡ ወልድ፡ አጽምኡ፡ ሕዝብየ፡ <gap reason="omitted" resp="DN"></gap></seg></q>
+
+                           </item>
+
+                           <item xml:id="a2">
+                              <locus target="#59ra" facs="059"/>
+                              <desc type="DonationNote">The name of the donor, <persName ref="PRS9451Tewoflos">Gabra Wald</persName>, is
```
Comment (thea-m) [inline ES/ESkae011.xml:297]: I think it's just the wrong ID for Gabra Wald, can you check throughout the manuscript as well?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, it will be very useful!

File: ES/ESkae011.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESkae011.xml]: corrected

File: ES/ESkae011.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESkae011.xml]: corrected

**Comment by DenisNosnitsin1970** [comment]
thanks!

**Pull request merged**